templates/api-blueprint.swig

```diff
@@ -79,6 +79,10 @@ FORMAT: 1A
 
 {{ transition.description }}
 
+{% for copy in transition|getCopy %}
```

Comment:
I don't think transactions can have descriptions in API Blueprint. Where would this go? Under the transition information like a relation?